Add Di's coaching guide

tannerwelsh opened the pull request:
Pulled https://www.gitbook.com/book/dianavashti/how-to-get-the-most-out-of-coaching/details into `Learning_Guide/Coaching`

When merged,

- [x] Update the coaching goal (135) with link to this guide

tannerwelsh committed:
Fix spelling of coach-queue

tannerwelsh committed:
Add Di's coaching guide

tannerwelsh commented:
\cc @carlabagdonas @deadlyicon

jaredatron commented on Learning_Guide/Coaching.md:
```diff
@@ -0,0 +1,61 @@
+
+_Original author: Diana Vashti_.
+
+As a coach, the feel of the week is a lot different than working on goals. You have no specs, no codebase, and instead of a team of 1 or 2, you are working with the whole guild. You may have back-to-back “coach-queue” appointments or most of an afternoon doing code reviews off “Prrr”. With the change of pace and scenery comes a new set of skills that are ideal for sharpening during a week of coac ...
```
they are on a project now and I think they have specs as of this week? @jrob8577 @shereefb 

The coaching goal currently links to [this other guide](https://dianavashti.gitbooks.io/how-to-get-the-most-out-of-coaching/content/) should we merge them?

tannerwelsh replied:
@deadlyicon I have a task to update the link in the coaching goal to this guide when this is merged (added to this PR description too)

jaredatron commented on Learning_Guide/Coaching.md:
```diff
@@ -0,0 +1,61 @@
+
+## How do I know if I am coaching?
+
+Once you make it to level 3 you will become eligible to coach. If it’s your first week at level 3 you will likely coach that week.
```
you know you're coaching if you are on goal 135 and jrob, the coaching coordinator, tells you you're coaching.

jaredatron commented on Learning_Guide/Coaching.md:
```diff
@@ -0,0 +1,61 @@
+
+Once you make it to level 3 you will become eligible to coach. If it’s your first week at level 3 you will likely coach that week.
+
+It is important that you do not vote for a goal if you are coaching so check with the coach team lead each week you are level 3 or above to determine if you are on coach-duty or not.
```
can they / should they vote for goal 135?

tannerwelsh replied:
I think so. Updating to:

> You know you're coaching if the coaching coordinator tells you you're coaching. You'll be asked to vote for [goal 135](http://jsdev.learnersguild.org/goals/135-Coaching.html).

jaredatron commented on Learning_Guide/Coaching.md:
```diff
@@ -0,0 +1,61 @@
+As a coach you have 3 main responsibilities:
+
+1. **Maintain available hours for coach-queue appointments**. These are half-hour blocks learners can schedule through the coach-queue app when they are feeling blocked. Your google calendar will be the source of your availability so make sure you block off time you do not have available for appointments there.
+1. **Complete code reviews from the Prrr app**. Make yourself available to answer questions learners may have regarding your review of their code/project.
```
we do not use Prrr anymore, Axe this part

jaredatron commented on Learning_Guide/Coaching.md:
```diff
@@ -0,0 +1,61 @@
+
+Check out the [Technical Coach](../Game_Manual/Levels_and_Roles.md#technical-coach) outline in [Levels and Roles](../Game_Manual/Levels_and_Roles.md).
+
+### Okay, but what does my day actually look like?
```
I think we should add something about standups here

tannerwelsh replied:
what needs to be said?

jrobcodes replied:
Add a responsibility perhaps, something along the lines of:
```md
As a coach you have 4 main responsibilities:

1. **Hold daily stand up meetings for your teams at 9:00am**.  These meetings should consist of every learner relating:
    1. What they did yesterday
    1. What they have planned for today
    1. What is blocking them from making progress
If a learner is blocked, you should request that they add a request to coach-queue, so that you can begin the day helping unblock your teams.
```

jrobcodes replied:
Oh nevermind, I didn't see the `join morning stand-ups` item lol.  Linear reading FTL. ☹️  I would expand on that with the above, and change the `Join` to `Hold` so that is sounds more like the responsibility of the coach to facilitate these.

jaredatron commented on Learning_Guide/Coaching.md:
```diff
@@ -0,0 +1,61 @@
+
+## Additional Tips
+
+- **Always prioritize participating in coaching sessions when you have escalated to an SEP**. This is one of the best opportunities to learn you may encounter. You attempted to solve a problem, and when you could not you get to immediately see how someone with a ton of experience solves the issue. If you have another appointment scheduled move it back so that you get to learn from the SEP.
```
**You must stay with the SEP when escalating so you too can learn the solution to the problem**

jaredatron commented on Learning_Guide/Coaching.md:
```diff
@@ -0,0 +1,61 @@
+
+- **Always prioritize participating in coaching sessions when you have escalated to an SEP**. This is one of the best opportunities to learn you may encounter. You attempted to solve a problem, and when you could not you get to immediately see how someone with a ton of experience solves the issue. If you have another appointment scheduled move it back so that you get to learn from the SEP.
+- **If a team does not know how to explain the situation to a degree you need to be helpful, focus on coaching them towards breaking down the problem into a more manageable chunk where they can describe the issue**. Three questions to consider are: What do you think the issue is? What have you tried so far? What would you expect if it worked?
+- **If you know you will need your regular afternoon coffee break, schedule it ahead of time, you can get booked up quickly.**
```
There is no way to schedule this. Just take breaks when you need to

jaredatron commented on Learning_Guide/Coaching.md:
```diff
@@ -0,0 +1,61 @@
+- **Always prioritize participating in coaching sessions when you have escalated to an SEP**. This is one of the best opportunities to learn you may encounter. You attempted to solve a problem, and when you could not you get to immediately see how someone with a ton of experience solves the issue. If you have another appointment scheduled move it back so that you get to learn from the SEP.
+- **If a team does not know how to explain the situation to a degree you need to be helpful, focus on coaching them towards breaking down the problem into a more manageable chunk where they can describe the issue**. Three questions to consider are: What do you think the issue is? What have you tried so far? What would you expect if it worked?
+- **If you know you will need your regular afternoon coffee break, schedule it ahead of time, you can get booked up quickly.**
+- **Ask for SEPs or other coaches to review your code reviews.**
```
I dont understand this one. No one does this.

jaredatron commented on Learning_Guide/Coaching.md:
```diff
@@ -0,0 +1,61 @@
+- **If a team does not know how to explain the situation to a degree you need to be helpful, focus on coaching them towards breaking down the problem into a more manageable chunk where they can describe the issue**. Three questions to consider are: What do you think the issue is? What have you tried so far? What would you expect if it worked?
+- **If you know you will need your regular afternoon coffee break, schedule it ahead of time, you can get booked up quickly.**
+- **Ask for SEPs or other coaches to review your code reviews.**
+- **Review code reviews, see what other folks are catching that you may not be.**
```
The work review implies a `code review` and we don't ask learners to do code reviews anymore. We ask them to coach and we ask them to do "goal completion evaluations" AKA `/review`.

jaredatron commented on Learning_Guide/Coaching.md:
```diff
@@ -0,0 +1,61 @@
+- **If you know you will need your regular afternoon coffee break, schedule it ahead of time, you can get booked up quickly.**
+- **Ask for SEPs or other coaches to review your code reviews.**
+- **Review code reviews, see what other folks are catching that you may not be.**
+- **If you know right away you cannot un-block a block in 30 minutes, escalate ASAP and make sure to watch the SEP unblock the learner/team.**
```
redundant

jaredatron commented on Learning_Guide/Onboarding.md:
```diff
@@ -16,7 +16,7 @@ These documents contain the latest and greatest information about our Learning O
 * [Coach Que](https://coach-queue.apps.learnersguild.org/): Pronounced "coach queue", our internal system for booking coaching sessions with more advanced learners. You'll need to be logged into Echo before you can start using Coach Que.
 
-Fun fact: Prrr and Coach-Que were built and are maintained by current Guild members.
+Fun fact: Prrr and Coach-Queue were built and are maintained by current Guild members.
```
remove Prrr

jaredatron commented on Learning_Guide/Coaching.md:
```diff
@@ -0,0 +1,61 @@
+
+**Tip**: It is a good practice to block off time at the end and beginning of each day to focus on code reviews, and to keep as much middle-of-the-day time as possible available for coach-queue appointments.
+
+## What relevant skills does coaching teach me?
```
This whole section is weird. A lot of odd questions. Maybe we can re-word this to be in the tips section?

tannerwelsh committed:
Integreate most of @deadlyicon's feedback

tannerwelsh commented:
@deadlyicon most changes added. Let me know if you think this is ready enough to merge.

jrobcodes commented on Learning_Guide/Coaching.md:
```diff
@@ -0,0 +1,51 @@
+
+_Original author: Diana Vashti. Thanks Di!_
+
+As a coach, the feel of the week is a lot different than working on goals. You have no specs, no codebase, and instead of a team of 1 or 2, you are working with the whole guild. You may have back-to-back “coach-queue” appointments. With the change of pace and scenery comes a new set of skills that are ideal for sharpening during a week of coaching.
```
```
You may have back-to-back "coach-queue" requests.
```
 instead of
 ```
You may have back-to-back "coach-queue" appointments.
```

jrobcodes commented on Learning_Guide/Coaching.md:
```diff
@@ -0,0 +1,51 @@
+
+1. **Maintain available hours for coach-queue appointments**. These are half-hour blocks learners can schedule through the coach-queue app when they are feeling blocked. Your google calendar will be the source of your availability so make sure you block off time you do not have available for appointments there.
+1. **Join morning stand-ups**.
+1. **Escalating code blocks and issues to SEPs if you are unable to resolve the learners block during your appointment** (ideally, if after 15 minutes you do not feel you are on a solid path to debugging the issue, a coach should find an available SEP to bring in.)
```
Escalating code blocks and issues to SEPs *__or fellow coaches__* if you are unable...

jrobcodes commented on Learning_Guide/Coaching.md:
```diff
@@ -0,0 +1,51 @@
+1. **Join morning stand-ups**.
+1. **Escalating code blocks and issues to SEPs if you are unable to resolve the learners block during your appointment** (ideally, if after 15 minutes you do not feel you are on a solid path to debugging the issue, a coach should find an available SEP to bring in.)
+
+**Tip**: It is a good practice to block off time at the end and beginning of each day to focus on code reviews, and to keep as much middle-of-the-day time as possible available for coach-queue appointments.
```
Is this tip necessary any longer?  This was relevant when coaches were assigned appointments for each request...

tannerwelsh replied:
Removing

jrobcodes commented on Learning_Guide/Coaching.md:
```diff
@@ -0,0 +1,51 @@
+
+## Additional Tips
+
+- **You must stay with the SEP when escalating so you too can learn the solution to the problem**. This is one of the best opportunities to learn you may encounter. You attempted to solve a problem, and when you could not you get to immediately see how someone with a ton of experience solves the issue. If you have another appointment scheduled move it back so that you get to learn from the SEP.
```
We can remove the last sentence - appointments are no longer scheduled.

tannerwelsh committed:
Integrate @jrob8577's feedback; remove "appointments"

Change "appointments" -> "sessions"

tannerwelsh commented:
Thanks @deadlyicon and @jrob8577! Integrated some more changes, and changed the "appointments" language to "requests/sessions". Also changed the reference to keeping GCal open, and instead just advised to "keep hours available"
